contrib/graph-gophers/graphql-go/option.go

```diff
@@ -0,0 +1,17 @@
+package graphql // import "gopkg.in/DataDog/dd-trace-go.v1/contrib/graph-gophers/graphql-go"
```

Comment:
I forgot about the package level documentation. Can you please add that and give it a shot on `godoc`? It might be more intuitive to use the file `graphql.go`. We might want to move the canonical import path comment there too as it might be the file where people will look intuitively to find these headers.